sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaUnboundedReader.java

```diff
@@ -302,13 +303,11 @@ public Instant getCurrentTimestamp() throws NoSuchElementException {
   @Override
   public long getSplitBacklogBytes() {
     long backlogBytes = 0;
-
     for (PartitionState<K, V> p : partitionStates) {
       long pBacklog = p.approxBacklogInBytes();
       if (pBacklog == UnboundedReader.BACKLOG_UNKNOWN) {
         return UnboundedReader.BACKLOG_UNKNOWN;
       }
-      backlogBytes += pBacklog;
     }
 
     return backlogBytes;
```

Comment:
This method should remain as it is, right? `getSplitBacklogBytes` and may affect autoscaling behavior according to the docs and this change would have this override return either 0 or `UnboundedReader.BACKLOG_UNKNOWN` instead of the split's approximate backlog bytes across assigned partitions or `UnboundedReader.BACKLOG_UNKNOWN`.

Reply:
yes I agree,  this was deleted accidentally.  Reverted that.